src/test/java/uk/gov/pay/adminusers/service/UserServicesTest.java

```diff
@@ -79,6 +83,25 @@ public void shouldFindAUserByExternalId() throws Exception {
     }
 
     @Test
+    public void shouldFindAUsersByExternalIds() throws Exception {
+        User user1 = aUser();
+        User user2 = anotherUser();
+
+        UserEntity userEntity1 = aUserEntityWithTrimmings(user1);
+        UserEntity userEntity2 = aUserEntityWithTrimmings(user2);
+
+        List<String> userExternalIds = Arrays.asList(user1.getExternalId(), user2.getExternalId());
+        List<UserEntity> userEntities = Arrays.asList(userEntity1, userEntity2);
+        when(userDao.findByExternalIds(userExternalIds)).thenReturn(userEntities);
```

Comment:
I would inline the previous two lines into this one:

```
when(userDao.findByExternalIds(Arrays.asList(user1.getExternalId(), user2.getExternalId()))).thenReturn(Arrays.asList(userEntity1, userEntity2));
```

Reply:
👍